docs/quickstart.html

```diff
@@ -304,15 +304,24 @@
 </p>
 
 <pre>
-&gt; <b>cat /tmp/file-input.txt | ./bin/kafka-console-producer --broker-list localhost:9092 --topic streams-file-input</b>
+&gt; <b>bin/kafka-topics.sh --create \</b>
```

Comment:
In the quick start section, it seems the topic names are emphasized inconsistently, sometimes they are double-quoted, e.g. `Let's create a topic named "test" with a single partition `, and sometimes they use `<pre>` and sometimes they use `<b>`, from multiple browsers I feel it's best be just double quoted or being bold (in Chrome `bold` is not very obvious though). But now I feel it may be quite out of the scope of this PR, so we can tackle it in another one.